jax/lax/lax.py

```diff
@@ -2309,20 +2309,11 @@ def _conv_general_dilated_shape_rule(
            "must be a positive integer, got {}.")
     raise ValueError(msg.format(batch_group_count))
   lhs_batch_count = lhs.shape[dimension_numbers.lhs_spec[0]]
+  
   if lhs_batch_count % batch_group_count != 0:
     msg = ("conv_general_dilated batch_group_count must divide lhs batch "
            "dimension size, but {} does not divide {}.")
     raise ValueError(msg.format(batch_group_count, lhs_batch_count))
-  if rhs.shape[dimension_numbers.rhs_spec[0]] % feature_group_count:
-    msg = ("conv_general_dilated rhs output feature dimension size must be a "
-           "multiple of batch_group_count, but {} is not a multiple of {}.")
-    raise ValueError(msg.format(rhs.shape[dimension_numbers.rhs_spec[0]],
-                                batch_group_count))
-
-  if not batch_group_count > 0 and feature_group_count > 0:
```

Comment:
This check is still needed and I think its intent is clear (what the mssage says is correct), but it probably should be checking `> 1` not `> 0` in the `if` condition.

Reply:
So to be clear, the condition should be ``if (batch_group_count > 1) and (feature_group_count > 1)``?

Reply:
I think so, yes.

Reply:
Done.